docs/guides/tutorials/getting_started.rst

```diff
@@ -6,119 +6,16 @@ Getting started with Great Expectations
 
 Welcome to Great Expectations! This tutorial will help you set up your first local deployment of Great Expectations that contains a small Expectation Suite to validate some sample data. We'll also introduce important concepts, with links to detailed material you can dig into later.
 
-The tutorial will walk you throw the following steps - **scroll to the bottom of this page and click the "Next" button to jump to the start of the tutorial!**
+The tutorial will walk you throw the following steps:
```

Comment:
Yup, I did it. I removed it the big numbers. People spent way too long reading this, but it didn't seem to really tell them anything.